front-end/src/components/Navbar.vue

```diff
@@ -1,13 +1,101 @@
-<script setup>
-
+<script>
+export default {
```

Comment:
This syntax is for older versions of vue. It still works, but to be consistent could you refactor this to use the `<script setup>` Composition API syntax documented here:
https://vuejs.org/guide/extras/composition-api-faq.html#what-is-composition-api 

You can also see examples in the MultipleChoiceQuiz.vue file

Reply:
Made the requested changes.